src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import java.util.Scanner;
+
+
+public class Duke {
+
+    private static Storage storage;
+    private static TaskList taskList;
+    private static Ui ui;
+
+    public static void main(String[] args) {
+
+        ui = new Ui();
+
+        ui.printGreeting();
+        storage = new Storage("./src/main/KiwiList.txt");
+
+        taskList = storage.load();
+
+        taskList.printList();
+
+        run(taskList, ui, storage);
+
+    }
+
+    public static void run(TaskList taskList, Ui ui, Storage storage) {
```

Comment:
While there are JavaDocs for most of the methods, it might be helpful to add JavaDocs descriptor for more methods/classes!

Reply:
Noted on that, thank you 👍